pvl-tools/pvl.cson

```diff
@@ -99,12 +99,11 @@ services:
 
     # This query operates on the newly extractaparsed comment, not the original page load
     # This is the selector for the post attachment links, which contains the
-    # proof text. It's the "<a> tags inside the div that's the immediate
-    # *sibling* of the 'userContet' div". The second of these three <a> tags
-    # contains the proof text, the others are blank. But we just check their concatenation.
+    # proof text. It's the second <a> tag inside the div that's the immediate
+    # *sibling* of the 'userContet' div. The second of these three <a> tags
+    # contains the proof text, the others are blank. We check only the second.
```

Comment:
The "others are blank" comment is now out of date. Can we update it? In this particular case, I think there ends up being more than three <a> tags :p